wiki/Difficulties/osu!taiko/en.md

```diff
@@ -0,0 +1,10 @@
+# osu!taiko
+
+There are the five difficulties a [beatmapper](/wiki/beatmapper) can [map](/wiki/map) for an osu!taiko [mapset](/wiki/mapset).
+The articles below explains how to set your beatmap up for a specific difficulty and what you should be aware of when making them.
```

Comment:
ditto, dood.
``The articles below explains how to prepare a beatmap for a specific difficulty and what to be aware of in the process of making the beatmap.``